libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -683,64 +609,39 @@ struct mrsRemoteVideoTrackConfig {
   const char* track_name{};
 };
 
-enum class mrsDataChannelConfigFlags : uint32_t {
-  kOrdered = 0x1,
-  kReliable = 0x2,
-};
-
-inline mrsDataChannelConfigFlags operator|(
-    mrsDataChannelConfigFlags a,
-    mrsDataChannelConfigFlags b) noexcept {
-  return (mrsDataChannelConfigFlags)((uint32_t)a | (uint32_t)b);
-}
-
-inline uint32_t operator&(mrsDataChannelConfigFlags a,
-                          mrsDataChannelConfigFlags b) noexcept {
-  return ((uint32_t)a | (uint32_t)b);
-}
-
 struct mrsDataChannelConfig {
-  int32_t id = -1;      // -1 for auto; >=0 for negotiated
-  const char* label{};  // optional; can be null or empty string
+  int32_t id = -1;  // -1 for auto; >=0 for negotiated
   mrsDataChannelConfigFlags flags{};
+  const char* label{};  // optional; can be null or empty string
 };
 
-struct mrsDataChannelCallbacks {
-  mrsDataChannelMessageCallback message_callback{};
-  void* message_user_data{};
-  mrsDataChannelBufferingCallback buffering_callback{};
-  void* buffering_user_data{};
-  mrsDataChannelStateCallback state_callback{};
-  void* state_user_data{};
-};
-
-/// Add a new data channel.
-/// This function has two distinct uses:
-/// - If id < 0, then it adds a new in-band data channel with an ID that will be
-/// selected by the WebRTC implementation itself, and will be available later.
-/// In that case the channel is announced to the remote peer for it to create a
-/// channel with the same ID.
-/// - If id >= 0, then it adds a new out-of-band negotiated channel with the
-/// given ID, and it is the responsibility of the app to create a channel with
-/// the same ID on the remote peer to be able to use the channel.
+/// Add a new data channel to a peer connection.
+///
+/// The initial configuration of the data channel is provided by |config|, and
+/// is mandatory. The caller can optionally provide a set of callbacks to
+/// register with the data channel via the |callbacks| optional argument. The
```

Comment:
There is no `callbacks`! Maybe this is meant to point to mrsDataChannelRegisterCallbacks?

Reply:
Ah yes that comment is stale, callbacks have been removed from the creation method, and are not registered separately.

Reply:
Fixed.